core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -941,10 +939,10 @@ public static String getDefaultOncoprintView() {
     
     public static boolean useCustomDriverAnnotations() {
         String useCustomDriverAnnotations = properties.getProperty(USE_CUSTOM_DRIVER_ANNOTATIONS);
-        boolean showBinaryCustomDriverAnnotation = showBinaryCustomDriverAnnotation();
-        boolean showClassCustomDriverAnnotation = showClassCustomDriverAnnotation();
+        String getBinaryCustomDriverAnnotationMenuLabel = getBinaryCustomDriverAnnotationMenuLabel();
+        String getClassCustomDriverAnnotationMenuLabel = getClassCustomDriverAnnotationMenuLabel();
         if (useCustomDriverAnnotations == null || useCustomDriverAnnotations.isEmpty() 
-            || !(showBinaryCustomDriverAnnotation || showClassCustomDriverAnnotation))
+            || (getBinaryCustomDriverAnnotationMenuLabel==null || getClassCustomDriverAnnotationMenuLabel==null))
```

Comment:
I think you will need to compare to empty string as well, since the variable could be there, but just set to nothing. E.g. 
```
oncoprint.custom_driver_annotation.class.menu_label=
```